2019-08-10 08:56:41 jdbc[5]: null
org.h2.message.DbException: アプリケーションにより閉じられていない接続がガベージコレクトされました
The connection was not closed by the application and is garbage collected [90018-199]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:242)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:182)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:299)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:782)
	at org.h2.server.web.WebApp.login(WebApp.java:1004)
	at org.h2.server.web.WebApp.process(WebApp.java:219)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:169)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.base/java.lang.Thread.run(Thread.java:844)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: アプリケーションにより閉じられていない接続がガベージコレクトされました
The connection was not closed by the application and is garbage collected [90018-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 15 more
